src/data/postgres/tables/device.test.ts

```diff
@@ -57,6 +57,8 @@ describe('device tests', () => {
       connectionstatus as "connectionStatus",
       mpsusername as "mpsusername",
       tenantid as "tenantId"
+      friendlyhostname as "friendlyhostname"
```

Comment:
these tests don't look correct - they are missing commas between the columns in the query

Reply:
updated